webapi/Extensions/ServiceExtensions.cs

```diff
@@ -123,30 +123,31 @@ internal static IServiceCollection AddCorsPolicy(this IServiceCollection service
     /// <exception cref="InvalidOperationException"></exception>
     public static IServiceCollection AddPersistentOcrSupport(this IServiceCollection services)
```

Comment:
Why is OCR support still needed if the memory service handles all image processing?

Reply:
Its not, I just haven't ripped it all out yet since it wasn't clear how controversial moving OCR to semantic-memory might be.

https://github.com/microsoft/semantic-memory/pull/29

I hope to get this integrated today or Monday.